array-manipulation-2/src/take-right.js

```diff
@@ -0,0 +1,14 @@
+/* exported takeRight */
+function takeRight(array, count) {
+  var arr = [];
+  for (var i = 0; i < array.length; i++) {
```

Comment:
Instead of starting at the beginning, try looping backwards. you shouldn't need the `if/else` statements at all